### PR TITLE
Add button role to improve accessibility in common/directives #4111

### PR DESCRIPTION
**Pull request by onyeepeace**
This pull request makes the following changes:
- Adds button role to improve accessibility

Testing checklist:
- [x] The buttons which are not button elements now act as buttons

- [x] I certify that I ran my checklist

Fixes ushahidi/platform#4111.

Ping @ushahidi/platform

### CODE REVIEW

File: app/common/directives/first-time-config.html [hunk 1]
```diff
@@ -48,13 +48,13 @@ <h1 translate>settings.customize_your_new_deployment</h1>
...
         </a>
         <div class="divider padded"></div>
         <p translate>settings.invite_people</p>
-        <a class="button button-primary" ui-sref="settings.users" ng-click="closeModal()">
+        <a class="button button-primary" role="button" ui-sref="settings.users" ng-click="closeModal()">
             <span translate>settings.add_new_users</span>
         </a>
     </div>
-</modal>
+</modal>
```
Comment (renujain31) [inline app/common/directives/first-time-config.html:60]: Hey! Can you remove this change. Just add a newline after `</modal>` and it should be fine.

**Review by renujain31 — CHANGES_REQUESTED** [review]
Hey @onyeepeace! It looks good to me just one change.

**Review by renujain31 — APPROVED** [review]
Hey, now its perfectly good 🎉 

Thanks @onyeepeace for your contribution.

**Comment by onyeepeace** [comment]
Thank you

**Review by rowasc — APPROVED** [review]
Brilliant, thank you ! @Obadha2 please test when you get a chance!

**Comment by Obadha2** [comment]
QA'd. passes. 👍

**Pull request merged**